packages/components/src/globals/scss/__tests__/__snapshots__/css--font-face-test.js.snap

```diff
@@ -4,680 +4,1634 @@ exports[`_css--font-face.scss experimental should output @font-face blocks from
 "@font-face {
   font-display: auto;
   font-family: 'IBM Plex Mono';
+  font-style: normal;
+  font-weight: 700;
+  src: local(\\"IBM Plex Mono Bold\\"), local(\\"IBMPlexMono-Bold\\"), url(\\"https://1.www.s81c.com/common/carbon/plex/fonts/IBM-Plex-Mono/fonts/complete/woff2/IBMPlexMono-Bold.woff2\\") format(\\"woff2\\"), url(\\"https://1.www.s81c.com/common/carbon/plex/fonts/IBM-Plex-Mono/fonts/complete/woff/IBMPlexMono-Bold.woff\\") format(\\"woff\\"); }
```

Comment:
Not quite sure why this was committed, I'll remove. It should only be in `v10` branch for now